app/routes/application.js

```diff
@@ -106,6 +106,10 @@ let ApplicationRoute = Route.extend(ApplicationRouteMixin, ModalHelper, SetupUse
   afterModel() {
     set(this.controllerFor('navigation'), 'allowSearch', false);
     $('#apploading').remove();
+    this.get('config.configDB').get('current_user').then((user) => {
+      let language = user.i18n || 'en';
```

Comment:
When you call `configDb.get('current_user')` it pulls a record with an `_id === 'current_user'`, but when you save the record with `user._id` the id of that record is going to be something like `org.couchdb.user:couchadmin`, not `current_user`.  Does that make sense?